operations/research-and-development/product/release-process/mobile-release.md

```diff
@@ -8,135 +8,12 @@ Please refer to [the Mobile Release Playbook](https://community.mattermost.com/p
-
-### F. \(T-minus 2 working days\) Release Build Cut
-
-The final release is cut. If an urgent and important issue needs to be addressed between major releases, a bug fix release \(e.g., 1.1.1\) may be created.
-
-1. Release Manager:
-   * Post this checklist in Release Checklist channel
-   * Verify all items in the last posted release checklist are complete
-   * Submit Changelog PR for review
-     * Merge Changelog PR after review is complete
-2. Build: 
-   * Tags the release and cuts the final build
-   * Upload the `.ipa` and `.apk` files to GitHub
-3. Marketing:
-   * Finish draft of marketing images and send to marketing lead for review
-4. Docs:
-   * Finalize docs
-     * If reviews are not complete, hold a 30-minute doc review meeting with PMs and anyone else who has changed or reviewed docs this release and wants to join
-     * Submit a correction PR for any incorrect formatting or other errors missed during the initial review
-
-### G. \(T-minus 0 working days\) Release Day and Build Submitted to App Stores
-
-1. Release Manager:
-   * Post this checklist in Release Checklist channel
-   * Verify all items in the last posted release checklist are complete
-   * Submit final build to Google Play Store and iTunes Connect
-   * Confirm that minimum server version required is clear in update notes in the App stores and in GitHub README and Releases pages
-   * Close the release in Jira
-   * Schedule a release retrospective meeting, to be held within five days from the release 
-   * Post key dates for the next release in the header of the Native Mobile Apps channel and remove links to RC candidates and testing spreadsheet
-     * Make sure that statutory holidays for Canada and US are accounted for in the release dates
-   * Check for any [UserVoice](https://docs.google.com/spreadsheets/d/1nljd4cFh-9MXF4DxlUnC8b6bdqijkvi8KHquOmK8M6E/edit#gid=0) feature suggestions that were completed in the current release
-2. Build:
-   * Merge the release branch back in to master
-   * Review and update project dependencies as needed
-   * Create release in GitHub
-
-### H. \(T-plus X days\) Release Marketing
-
-1. Marketing:
-   * After the apps are approved and on the App Store \(number of days may vary\), send out release marketing
-     * Update the app version on the [download page](https://mattermost.com/download/)
-     * Send out a Twitter announcement
-   * Double check that minimum server version required is clear in update notes in the App stores and in GitHub README and Releases pages
-
+Note: T-minus counts are measured in "working days" \(weekdays, Monday through Friday, excluding [the listed statutory holidays](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/paid-time-off#holidays)\) prior to release day.
+
+**Schedule for Mobile App releases**:
+ - Feature Complete deadline is approximately 1 month prior to the release day.
+ - Cut release branch based off `main` and cut beta builds (around T-14).
```

Comment:
I would add here to cut an RC based on the release branch and start testing. here is the reason

1. the weekly beta builds are always based of main branch and can be used for progressing testing until we branch for release, once branched, beta could have code that is not going to be part of the release.
2. Give more time to QA to run their test suite (automated and manual)
3. Identify problems with the release early

Thoughts @saturninoabril

Reply:
I agree with this suggestion, I'll also add @larkox to share any additional thoughts.

Reply:
Thanks @enahum and I totally agree. I honestly thought that we normally do progressive testing during release period for mobile app. I didn't realize that "the weekly beta builds are always based of main branch" is the case.

Reply:
Thanks, I also agree, and apparently had confusion as well. :)